lecture00-introduction/introduction.tex

```diff
@@ -153,7 +153,9 @@ \section{Who we are}
 \begin{frame}{Who we are}
     Johannes Schmidt <johannes.schmidt@boku.ac.at>\\
     \begin{itemize}
-        \item
+        \item Associate Prof. in Energy \& Resource Economics at the Institute for Sustainable Economic Development
+        \item Works on modeling of renewable energy systems in R, GAMS (and python)
```

Comment:
Python is usually spelled "Python" AFAIK

Reply:
```suggestion
        \item Works on modeling of renewable energy systems in R, GAMS (and Python)
```